docker/docker-compose.yml

```diff
@@ -111,7 +111,7 @@ services:
   controller-dev:
     image: vc-authn-oidc-controller-dev
     ports:
-      - "${IP:-0.0.0.0}:5000:5000"
```

Comment:
The right-hand port should be left as before, as it is internal to the Docker image and not mapped to the host.